packages/core/src/components/Dropdown/Dropdown.test.jsx

```diff
@@ -40,7 +40,7 @@ function render(customProps = {}, optionsCount = 1, deep = false) {
 
 describe('Dropdown', () => {
   it('renders a select menu', () => {
-    const data = render({ value: '1' });
+    const data = render({ value: '1', ariaLabel: 'test aria label' });
```

Comment:
@scitech I like this approach to adding the `aria-label` explicitly. Is there a way we can check for a `FormLabel` and throw a warning if the component has both present? Asking because an aria-label will override the label on the page, and that's almost never what we want. Form input labels are ideally written descriptively for all users.

My recommendation is to warn/error on this state if possible, or if not, note this as a warning in the component documentation.

Any further thoughts @davidakennedy ?

Reply:
Hey @1Copenut, for my own clarification, are you saying that `aria-label` should be used only if there isn't a `formLabel` in use?

Reply:
Correct. We should not use an aria-label if we already have a `formLabel` associated with the select. The `formLabel` is the preferred means to identify a form input/select, but we can and should use the aria-label in cases like a checked child.

Reply:
@1Copenut My last commit adds some documentation and a console warning!